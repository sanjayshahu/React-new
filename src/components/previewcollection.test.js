import { shallow,mount,render } from "enzyme";
import { idText } from "typescript";
import {React}from "react";
import PreviewCollection from "./previewcollection";
import toJson from 'enzyme-to-json';
it('take snapshot',()=>{
    const temp_DATA = [
        {
          id: 1,
          title: 'Hats',
          routeName: 'hats',
          items: [
            {
              id: 1,
              name: 'Brown Brim',
              imageUrl: 'https://i.ibb.co/ZYW3VTp/brown-brim.png',
              price: 25
            }
        ]
    }]//here props were mandatory for previewcollection

    expect( toJson(shallow(<PreviewCollection title={temp_DATA[0].title} items={temp_DATA[0].items}/>))).toMatchSnapshot();//use this command for taking snapshot
}) 
