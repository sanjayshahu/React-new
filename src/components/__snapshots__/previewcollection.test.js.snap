// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`take snapshot 1`] = `
<div
  className="collection-preview"
>
  <h1
    className="title"
  >
    HATS
  </h1>
  <div
    className="preview"
  >
    <PreviewCollectionItem
      imageUrl="https://i.ibb.co/ZYW3VTp/brown-brim.png"
      key="1"
      name="Brown Brim"
      price={25}
    />
  </div>
</div>
`;
